### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by jakubkoci** [openwallet-foundation/credo-ts#430] [Architecture & Design]
# Roadmap

## 0.1.0

- #349

## AIP 1.0

- #58
- #112

## AIP 2.0

All entries for AIP 1.0 are also required for AIP 2.0

### Base Requirements

- #344 (@jakubkoci)
- #345 (@jakubkoci)
- #493
- #352 
- #353 
- #495
- #497
- #498
- RFC 0519: Goal Codes (will be handled by respective protocols)

### Mediator Subtarget

- Complete

### Indy Based Credentials

- #357
- #494

### JSON-LD Based Credentials

Doesn't include support for specific signature suites yet.

- #356 
- #355
- #354 

### BBS+ Based Credentials

Needs "JSON-LD Based Credentials" subtarget to support this target

- #496

### DIDComm V2 Prep

- Add support for RFC 0587: Encryption Envelope V2

### Chat Related Features

- Complete

**Comment by TimoGlastra** [comment]
Closing, all issues addressed
